src/build/env.py

```diff
@@ -289,6 +289,9 @@ def _create_isolated_env_venv(path: str) -> Tuple[str, str]:
     return executable, script_dir
 
 
+_sysconfig_get_paths = sysconfig.get_paths
```

Comment:
As somebody who reads this code, I believe this assignment would benefit from a comment.